IEWebkitImpl/Browser.ts

```diff
@@ -161,7 +163,9 @@ module F12.Proxy {
                         case "Page":
                             this.ProcessPage(methodParts[1], request);
                             break;
-
+                        case "DOM":
+                            domHandler.ProcessDOM(methodParts[1], request);
```

Comment:
Does domHandler need to be global? Couldn't it just be constructed as a member to this class?